Add "create project" docs for cookieplone (#1714)
plone/documentation: docs/glossary.md

    A command-line utility that creates projects from cookiecutters (project templates), for example, creating a Python package project from a Python package project template.
    [See Cookiecutter's documentation](https://cookiecutter.readthedocs.io/en/stable/).

Cookieplone
    ```{versionadded} Volto 18.0.0-alpha.43
    ```

    [Cookieplone](https://github.com/plone/cookieplone) is the method to create a Plone project.

>     You can use Cookieplone to build a backend add-on, a new Volto add-on, a full project with both backend and frontend, or even documentation.

Author: ```suggestion
    You can use Cookieplone to build a backend add-on, a new Volto add-on, or a full project with both backend and frontend.
```

(I think adding a documentation template is a great idea, but it doesn't exist yet.)

Reviewer: I pulled that from its readme. See https://github.com/plone/cookieplone/pull/40

    Cookieplone simplifies the process using robust Cookiecutter templates from {term}`cookieplone-templates`.
    
cookieplone-templates

>     [`cookieplone-templates`](https://github.com/plone/cookieplone-templates) is a collection of templates for Plone integrators to create web application projects.

Author: ```suggestion
    [`cookieplone-templates`](https://github.com/plone/cookieplone-templates) is a collection of templates used by {term}`Cookieplone`.
```

(It's really a part of Cookieplone, not something to use directly.)

Reviewer: I pulled that from its [readme](https://github.com/plone/cookieplone-templates?tab=readme-ov-file#about).

> Powered by [Cookieplone](https://github.com/plone/cookieplone), this project is a collection of templates intended to be used by Plone integrators.

How should we align these two statements?

Author: @stevepiercy I already gave my suggestion. My problem with saying "intended to be used by Plone integrators" in this context is that it sounds like those integrators should go find cookieplone-templates and use it, but actually the correct way for them to do that is to run Cookieplone, which uses cookieplone-templates by default.

Reviewer: OK, I read the Cookieplone statement as the templates should be used directly by integrators. "Powered by Cookieplone" is marketing speak. I'll create a PR in Cookieplone instead and accept this change.
